source/fem-dem/vans_assemblers.cc

```diff
@@ -1407,7 +1407,7 @@ GLSVansAssemblerSaffmanMei<dim>::calculate_particle_fluid_interactions(
           // Saffman-Mei coefficient C_s
           alpha = 0.5 * particle_properties[DEM::PropertiesIndex::dp] /
                   relative_velocity[particle_number].norm() *
-                  (velocity_curls_3d[particle_number].norm() + 1e-9);
+                  (vorticity_3d[particle_number].norm() + 1e-9);
```

Comment:
1e-9 is a bit on the small side maybe 1e-12?

Reply:
Fixed.